### PR TITLE
Regression: Add tests to new banners REST endpoints

### PR DESCRIPTION
**Pull request by lucassartor**
## Proposed changes (including videos or screenshots)

Add tests for the new `banners.*` endpoints: `banners.getNew` and `banners.dismiss`.

### CODE REVIEW

File: tests/end-to-end/api/21-banners.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { expect } from 'chai';
+
+import { getCredentials, api, request, credentials } from '../../data/api-data.js';
+
+describe.only('banners', function() {
```
Comment (sampaiodiego) [inline tests/end-to-end/api/21-banners.js:5]:
```suggestion
describe('banners', function() {
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]
You forgot just a `.only` .. but everything else looks good! :D

**Comment by lucassartor** [comment]
> You forgot just a `.only` .. but everything else looks good! :D

Sorry! That was my bad, now it's fixed.

**Review by sampaiodiego — APPROVED** [review]

**Comment by sampaiodiego** [comment]
that's totally fine @lucassartor .. you did a good job, thx <3

**Pull request merged**